### ISSUE TITLE
about dataset  and training

### ISSUE THREAD

**Issue by xiaomingdaren123** [MarekKowalski/DeepAlignmentNetwork#34]
hi,MarekKowalski ，thank you for your code about deep alignment network.but i have some trouble about it ,i hope you could help me.
1、When I download the 300-W dataset based on the URL you gave and unzip the part2 of 300-W dataset,an error occurred .I don't know what's going on, I can't open the file, and I tried it many times without success.so ,Can you send Part2 data of 300-W to me, or can I use only  other data sets( LFPW, HELEN, AFW and IBUG)without 300-W data set?what should I do
2、How to train the first and second stage？ Can you tell me the training steps?
I am looking forward to your answer, thank you.

**Comment by MarekKowalski** [comment]
Hi,

Here are my answeres to your questions:
1. I cannot send you the 300-W dataset, as it does not belong to me, please contact the IBUG group at Imperial College. You can definitely train the model only with the other datasets, 300-W is only a test set, I did not use it for training.
2. To train the model you should:
 - extract the datasets to the ./data/images/ directory
 - run the TrainingSetPreparation.py and TestSetPreparation.py scripts
 - use the DANTraining.py script for training (the comments in the code tell you about how to do first and second stage).

Thanks,

Marek
